static/app/components/replays/replayHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import EventOrGroupTitle from 'sentry/components/eventOrGroupTitle';
+import EventMessage from 'sentry/components/events/eventMessage';
+import FeatureBadge from 'sentry/components/featureBadge';
+import space from 'sentry/styles/space';
+import {Event} from 'sentry/types/event';
+import {getMessage} from 'sentry/utils/events';
+
+const ReplayHeader = ({event}: {event: Event}) => {
```

Comment:
I would probably pull out the type here and consistently use `type Props = { ... }`